letter_project/src/components/MainPage/Letters.tsx

```diff
@@ -11,24 +11,25 @@ export default function Letters(){
         postId:number
     
       }
+    const navigate=useNavigate();
     const [letterList,setletterList]=useState<LetterType[]>([]);
     const getLetter=async():Promise<void>=>{
         const response=await axios.get("https://jsonplaceholder.typicode.com/comments");
         setletterList(response.data);
     }
-    const letterClicked=()=>{
-        
+    const letterClicked=(letter:LetterType):void=>{
+        console.log(letter);
+        navigate(`/${letter.id}`)
     }
 
     useEffect(()=>{
         getLetter();
     },[])
 return(
     <Styled.Letters>
-         {letterList===undefined ? <div>로딩중입니다..</div> : <>
-       {letterList.slice(0,3).map((index:any)=>(<Styled.Letter>{index.name}이가 작성...</Styled.Letter>))}</>}
+        {letterList===undefined ? <div>로딩중입니다..</div> : <>
+        {letterList.slice(0,3).map((index:LetterType)=>(<Styled.Letter onClick={()=>letterClicked(index)}>{index.name}이가 작성...</Styled.Letter>))}</>}
```

Comment:
```suggestion
         {!letterList? <div>로딩중입니다..</div> : <>
     {letterList.slice(0,3).map((letter:LetterType)=>(<Styled.Letter onClick={()=>letterClicked(letter)}>{letter.name}이가 작성...</Styled.Letter>))}</>}
```
index보다는! letter가 더 명확한 변수명 같아요!